Assets/Prefabs/Character/Character.prefab

```diff
@@ -275,15 +278,14 @@ MonoBehaviour:
   SyncPositionX: 1
   SyncPositionY: 1
   SyncPositionZ: 1
-  SyncRotAngleX: 1
+  SyncRotAngleX: 0
   SyncRotAngleY: 1
-  SyncRotAngleZ: 1
-  SyncScaleX: 1
-  SyncScaleY: 1
-  SyncScaleZ: 1
-  PositionThreshold: 0
-  RotAngleThreshold: 0
-  ScaleThreshold: 0
+  SyncRotAngleZ: 0
+  SyncScaleX: 0
+  SyncScaleY: 0
+  SyncScaleZ: 0
+  PositionThreshold: 0.001
```

Comment:
Did you test with higher values? This value is basically millimeters of precision, this seems quite a high precision.

Reply:
same with all others

Reply:
I did. I could increase this up to a certain point before it negatively impacted gameplay, but there were no bandwidth gains. However, when testing with the develop branch of NGO, the impact of thresholds was much higher. I would recommend that if we want to update those thresholds, we do so after those changes have been released and integrated into Boss Room

Reply:
could you test with that branch and check for those thresholds?
You'd see bandwidth gains in cases where there's no that much position or rotation changes (I'm thinking on those situations where imps gang up on you and push you slowly while you don't move yourself)